src/components/clusterConfiguration/DownloadIsoModal.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Modal, ModalVariant } from '@patternfly/react-core';
+import DownloadIso, { DownloadISOProps } from './DownloadIso';
+
+type DownloadIsoModalProps = DownloadISOProps & {
+  isOpen: boolean;
+};
+
+const DownloadIsoModal: React.FC<DownloadIsoModalProps> = ({ isOpen, ...props }) => (
```

Comment:
Related to https://github.com/rawagner/dynamic-cim/pull/4/files/7df6d034c9670fb111ca7d47b34c1635a6d10b90#r638496136 Should we use modals from assisted-ui-lib or handle it on the console side?

Reply:
Commented there and on the slack. IMO, it would bring unnecessary complexity to provide that feature for both modal and non-modal usa-cases. The outer context still drives modal-related primitives and visibility (isOpen, onClose).